db/Controllers/topics.controller.js

```diff
@@ -0,0 +1,17 @@
+
+const { fetchTopics } = require("../Models/models")
+
+const getTopics = (request, response, next)=>{
+
+    fetchTopics()
+    .then((topics)=>{ response.status(200)
+        .send({topics})
+     /* <--- passes err to app.js next? */
```

Comment:
This comment is similar to the one on line 3 in app.js, this is a comment which is for yourself rather than adding value to the code, so do remove them before submitting pull requests